<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>88</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>8</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="1" column="1" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2" >
     <widget class="QPushButton" name="pushButtonQuit" >
      <property name="text" >
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QPushButton" name="pushButtonStart" >
      <property name="text" >
       <string>What time is it? Click to update.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3" >
     <widget class="QLabel" name="labelTime" >
      <property name="text" >
       <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>(do not know yet)&lt;/p>&lt;/body>&lt;/html></string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
